torch/distributed/distributed_c10d.py

```diff
@@ -197,32 +197,80 @@ def __getattribute__(self, key):
 
 reduce_op = _reduce_op()
 
+# Not including _default_pg_init_method cuz it's dead code (nothing uses it in PT)
+class World:
+    def __init__(self):
+        self._default_pg = None
+        self._pg_map = {}
+        self._pg_names = {}
+        self._pg_group_ranks = {}
+        self._group_count = 0
+
+    @property
+    def default_pg(self):
```

Comment:
same, they would need docstrings if user-facing API

Reply:
Add docstring. It's a private method for now.